devsetup/scripts/gen-ansibleee-ssh-key.sh

```diff
@@ -18,42 +18,45 @@ set -ex
 # expect that the common.sh is in the same dir as the calling script
 SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 NAMESPACE=${NAMESPACE:-"openstack"}
-METADATA_NAME=${METADATA_NAME:-"ansibleee-ssh-key-secret"}
-OUTPUT_DIR=${OUTPUT_DIR:-"../out/edpm"}
+METADATA_NAME=${METADATA_NAME:-"dataplane-ansible-ssh-private-key-secret"}
+OUTPUT_DIR=${OUTPUT_DIR:-"../../out/edpm"}
 SSH_ALGORITHM=${SSH_ALGORITHM:-"rsa"}
 SSH_KEY_FILE=${SSH_KEY_FILE:-"ansibleee-ssh-key-id_rsa"}
 SSH_KEY_SIZE=${SSH_KEY_SIZE:-"4096"}
 
+pushd ${SCRIPTPATH}
+
 if [ ! -d ${OUTPUT_DIR} ]; then
     mkdir -p ${OUTPUT_DIR}
 fi
 
 pushd ${OUTPUT_DIR}
 
-if oc get secret ansibleee-ssh-key-secret 2>&1 1>/dev/null; then
-    echo "Secret ansibleee-ssh-key-secret already exists."
+if oc get secret dataplane-ansible-ssh-private-key-secret 2>&1 1>/dev/null; then
+    echo "Secret dataplane-ansible-ssh-private-key-secret already exists."
     echo "Delete it first to recreate:"
-    echo "oc delete secret ansibleee-ssh-key-secret"
+    echo "oc delete secret dataplane-ansible-ssh-private-key-secret"
     exit 0
 fi
 
 if [ ! -f ${SSH_KEY_FILE} ]; then
     ssh-keygen -f ${SSH_KEY_FILE} -N "" -t ${SSH_ALGORITHM} -b ${SSH_KEY_SIZE}
 fi
 
-cat <<EOF >ansibleee-ssh-key-secret.yaml
+cat <<EOF >dataplane-ansible-ssh-private-key-secret.yaml
 apiVersion: v1
 kind: Secret
 namespace: ${NAMESPACE}
 metadata:
     name: ${METADATA_NAME}
 data:
-    public_ssh_key: |
+    ssh_public_key: |
 $(cat ${SSH_KEY_FILE}.pub | base64 | sed 's/^/        /')
-    private_ssh_key: |
+    ssh_private_key: |
```

Comment:
if https://github.com/openstack-k8s-operators/dataplane-operator/pull/54 is merged first, we would need to change
```suggestion
    ssh-privatekey: |
```

Reply:
thanks, we can hold off on this PR until https://github.com/openstack-k8s-operators/dataplane-operator/pull/54 merges.